apps/src/code-studio/pd/application_dashboard/constants.js

```diff
@@ -81,21 +83,27 @@ export const ScholarshipStatusRequiredStatuses = [
  */
 export function getApplicationStatuses(type, addAutoEmail = true) {
   if (type === 'teacher') {
-    return {
-      unreviewed: 'Unreviewed',
-      incomplete: 'Incomplete',
-      pending: 'Pending',
-      waitlisted: `Waitlisted${autoEmailText(addAutoEmail)}`,
-      declined: `Declined${autoEmailText(addAutoEmail)}`,
-      accepted_not_notified: 'Accepted - Not notified',
-      accepted_notified_by_partner: 'Accepted - Notified by partner',
-      accepted_no_cost_registration: `Accepted - No cost registration${autoEmailText(
-        addAutoEmail
-      )}`,
-      registration_sent: `Registration Sent${autoEmailText(addAutoEmail)}`,
-      paid: 'Paid',
-      withdrawn: 'Withdrawn'
-    };
+    // [MEG] TODO: Once experiment is complete, remove omit
+    return omit(
```

Comment:
this is cool! i hadn't seen omit used before

Reply:
Yeah, I wasn't sure the best way to do this, and then lodash came in clutch